plugins/kotlin-dataframe/testData/box/groupBy_refine.kt

```diff
@@ -0,0 +1,12 @@
+import org.jetbrains.kotlinx.dataframe.*
+import org.jetbrains.kotlinx.dataframe.annotations.*
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.io.*
+
+
+fun box(): String {
+    val df = dataFrameOf("a", "b", "c")(1,2,3)
```

Comment:
at one point, some linter is going to complain about the lack of spaces after the comma's XD